ansible/secrets.example.yml

```diff
@@ -1,8 +1,17 @@
+# The ID of the GitHub app associated to the deployment
+github_app_id: 99999
+
 # The private key of the GitHub app associated to the deployment
-github_app_key: '-----BEGIN RSA PRIVATE KEY-----<base64_encoded_key>-----END RSA PRIVATE KEY-----'
+github_app_key: |-
+  -----BEGIN RSA PRIVATE KEY-----
+  <SECRET BASE46 GOES HERE>
```

Comment:
Nitpick: it should say BASE64 instead of BASE46

Reply:
Oops :laughing: Will fix.

Reply:
Fixed!